playbooks/cluster/openshift/config.yml

```diff
@@ -2,18 +2,57 @@
 - hosts: all
   remote_user: root
   pre_tasks:
+    - name: include_vars
+      include_vars:
+        file: "{{ playbook_dir }}/vars/{{ openshift_ver }}.yml"
+
+    - name: set SELinux to permissive mode
+      command: setenforce 1
+
+    - name: set SELinux to permissive mode under configuration file
+      selinux:
+        policy: targeted
+        state: enforcing
+
+    - name: stop and disable firewalld
+      register: result
+      service:
+        state: stopped
+        enabled: no
+        name: firewalld
+      failed_when: "result|failed and not 'Could not find the requested service' in result.msg"
+
+    - name: install epel repository
+      yum:
+        name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
I would use a `epel_repo_url` variable

Reply:
and I'm not sure about the syntax here, see: http://docs.ansible.com/ansible/latest/yum_repository_module.html#id3